### ISSUE TITLE
No stars shown on latest firefox

### ISSUE THREAD

**Issue by gh658804** [Stellarium/stellarium-web-engine#191]
https://stellarium-web.org/

The web user interface is shown but stars, landscape, constellations and in general everything else i turn on from the UI are not shown.

Using the latest Firefox (116.0.3) under Windows 10. I've also tried it in troubleshooting mode, it just does not work. In the same PC, it works fine in Chrome.

**Comment by gzotti** [comment]
Same version, Win11, no issue.

**Comment by gh658804** [comment]
I can't understand what's going on. I tried this from two different locations (work/home), with addons disabled and from within a Private window. Under Firefox, it just does not work. I even turned off the Antivirus program, nothing changed.

Anyone else has this problem?

**Comment by gzotti** [comment]
I have this difference with another program on a very old computer (2010 with only Intel HD graphics of first-generation Core-i7). OK on Chrome, no show of actual graphics on FF (GUI OK). The FF browser console or developer tools may at least give a hint on what goes wrong.

**Comment by gh658804** [comment]
> The FF browser console or developer tools may at least give a hint on what goes wrong.

Nice. I should have thought of it. Anyway, thanks for the tip. My laptop is not as old as yours, but it's a 5-6 yrs old model. Apart from Intel Graphics, it also has a Radeon 530 card. I tried disabling acceleration in FF but no luck...

In Console, i saw the following errors:

`Uncaught TypeError: Zn is undefined
    yn stellarium-web-engine.js:9
    wn stellarium-web-engine.js:9
    _core_render stellarium-web-engine.js:9
    a stellarium-web-engine.js:9
    i stellarium-web-engine.js:9
    onRuntimeInitialized stellarium-web-engine.js:9
    t stellarium-web-engine.js:9
    Ta stellarium-web-engine.js:9
    e stellarium-web-engine.js:9
    ot stellarium-web-engine.js:9
    t stellarium-web-engine.js:9
    n stellarium-web-engine.js:9
    promise callback*o/< stellarium-web-engine.js:9
    promise callback*o stellarium-web-engine.js:9
    gt stellarium-web-engine.js:9
    i stellarium-web-engine.js:9
    initStelWebEngine sw_helpers.js:33
    mounted App.vue:230
    promise callback*mounted App.vue:224
    VueJS 12
    56d7 main.js:152
    Webpack 6
stellarium-web-engine.js:9:74479
    yn stellarium-web-engine.js:9
    wn stellarium-web-engine.js:9
    <anonymous> stellarium-web-engine.9b8f0e47.wasm:107796
    <anonymous> stellarium-web-engine.9b8f0e47.wasm:386225
    <anonymous> stellarium-web-engine.9b8f0e47.wasm:618372
    _core_render stellarium-web-engine.js:9
    a stellarium-web-engine.js:9
    (Async: FrameRequestCallback)
    i stellarium-web-engine.js:9
    onRuntimeInitialized stellarium-web-engine.js:9
    t stellarium-web-engine.js:9
    Ta stellarium-web-engine.js:9
    e stellarium-web-engine.js:9
    ot stellarium-web-engine.js:9
    t stellarium-web-engine.js:9
    n stellarium-web-engine.js:9
    (Async: promise callback)
    o stellarium-web-engine.js:9
    (Async: promise callback)
    o stellarium-web-engine.js:9
    gt stellarium-web-engine.js:9
    i stellarium-web-engine.js:9
    initStelWebEngine sw_helpers.js:33
    mounted App.vue:230
    (Async: promise callback)
    mounted App.vue:224
    VueJS 12
    56d7 main.js:152
    Webpack 6
`

`Uncaught (in promise) RuntimeError: index out of bounds
    _core_add_font stellarium-web-engine.js:9
    le stellarium-web-engine.js:9
    setFont stellarium-web-engine.js:9
    promise callback*c074/</i</</c.setFont stellarium-web-engine.js:9
    mounted App.vue:236
    onReady sw_helpers.js:51
    onRuntimeInitialized stellarium-web-engine.js:9
    t stellarium-web-engine.js:9
    Ta stellarium-web-engine.js:9
    e stellarium-web-engine.js:9
    ot stellarium-web-engine.js:9
    t stellarium-web-engine.js:9
    n stellarium-web-engine.js:9
    promise callback*o/< stellarium-web-engine.js:9
    promise callback*o stellarium-web-engine.js:9
    gt stellarium-web-engine.js:9
    i stellarium-web-engine.js:9
    initStelWebEngine sw_helpers.js:33
    mounted App.vue:230
    promise callback*mounted App.vue:224
    VueJS 12
    56d7 main.js:152
    Webpack 6
stellarium-web-engine.9b8f0e47.wasm:430317:1
`

`Uncaught (in promise) RuntimeError: index out of bounds
    le stellarium-web-engine.js:9
    setFont stellarium-web-engine.js:9
    promise callback*c074/</i</</c.setFont stellarium-web-engine.js:9
    mounted App.vue:237
    onReady sw_helpers.js:51
    onRuntimeInitialized stellarium-web-engine.js:9
    t stellarium-web-engine.js:9
    Ta stellarium-web-engine.js:9
    e stellarium-web-engine.js:9
    ot stellarium-web-engine.js:9
    t stellarium-web-engine.js:9
    n stellarium-web-engine.js:9
    promise callback*o/< stellarium-web-engine.js:9
    promise callback*o stellarium-web-engine.js:9
    gt stellarium-web-engine.js:9
    i stellarium-web-engine.js:9
    initStelWebEngine sw_helpers.js:33
    mounted App.vue:230
    promise callback*mounted App.vue:224
    VueJS 12
    56d7 main.js:152
    Webpack 6
stellarium-web-engine.9b8f0e47.wasm:430317:1
`

**Comment by gzotti** [comment]
Yes, the "Exhausted GL driver options" are what also let my 2010 laptop fail on the graphics. I cannot help on Stellarium-web, would just use Chrome for now. But why should DISabling graphics acceleration solve a problem in graphics acceleration? Make sure the Radeon OpenGL drivers are OK, it works with FF, and FF is executed on the Radeon.

**Comment by gh658804** [comment]
> But why should DISabling graphics acceleration solve a problem in graphics acceleration?

Well, it's standard troubleshooting. It could perhaps display the missing parts (much slower though), so this could help determine if it's an issue with the driver.

> Make sure the Radeon OpenGL drivers are OK, it works with FF, and FF is executed on the Radeon.

Well, i have the latest driver updates for my laptop, so... I guess, that's it.
